cssFiles/button.css

```diff
@@ -179,4 +179,21 @@
     border-radius: 9999px;
     padding: 0;
     margin: 2rem;
+}
+
+/** links **/
+.linkp {
+    text-decoration: none;
+    padding: 1rem 2rem
+}
+
+.link-primaryp {
+    background-color: var(--primary-color);
+    border-radius: 0.5rem;
+    color: white;
+}
+
+.linkp:hover {
+    background-color:#002ead;
```

Comment:
If you're using following color multiple times, create a variable function for it